schema_source/aws_serverless_function.py

```diff
@@ -448,6 +448,7 @@ class ScheduleV2Event(BaseModel):
 KmsKeyArn = Optional[PassThroughProp]
 Layers = Optional[PassThroughProp]
 AutoPublishAlias = Optional[SamIntrinsicable[str]]
+AutoPublishAliasAdditionalProperties = Optional[SamIntrinsicable[bool]]
```

Comment:
Since it's now a boolean, perhaps `AutoPublishAliasAllProperties` or something similar would be clearer? Currently it's not obvious what "additional" refers to.

Reply:
Also same comment about intrinsics, this shouldn't support them explicitly.

Reply:
Yea I like `AutoPublishAliasAllProperties` more since they're not defining additional properties